sources/packages/backend/libs/integrations/src/sfas-integration/sims-sfas-files/sims-to-sfas-file-line-builder.ts

```diff
@@ -0,0 +1,90 @@
+import { StudentDetail } from "@sims/integrations/services/sfas";
+import { FixedFormatFileLine } from "@sims/integrations/services/ssh";
+import { SIMSToSFASRecordTypeCodes } from "../sfas-integration.models";
+import { SIMSToSFASHeader } from "./sims-to-sfas-header";
+import { SIMSToSFASStudentRecord } from "./sims-to-sfas-student-record";
+import { DisabilityStatus } from "@sims/sims-db";
+import { YNFlag } from "@sims/integrations/models";
+import { SIMSToSFASFooter } from "./sims-to-sfas.footer";
+
+export class SIMSToSFASFileLineBuilder {
+  private readonly fileLinesInternal: FixedFormatFileLine[] = [];
```

Comment:
I believe the object type should be `SIMSToSFASBaseRecord`.
For instance, the footer can get the total records by checking the record type.